proposals/rcp-51-dom-increment.md

```diff
@@ -0,0 +1,324 @@
+# RESO DOM Increment Endorsement
+
+| **RCP** | 51 |
+| :--- | :--- |
+| **Version** | **1.0.0** |
+| **Authors** | Bryan Burgers, Zenlist<br />Al McElmon, CoreLogic<br />Sergio Del Rio, T4BI |
```

Comment:
Usually these are emails so people can contact the spec authors if needed.

Reply:
Done.